corehq/apps/app_manager/models.py

```diff
@@ -5687,6 +5688,7 @@ def import_app(app_id_or_source, domain, source_properties=None, request=None):
     app = cls.from_source(source, domain)
     app.date_created = datetime.datetime.utcnow()
     app.cloudcare_enabled = domain_has_privilege(domain, privileges.CLOUDCARE)
+    app.progenitor_app_id = progenitor_app_id
```

Comment:
I think this also needs to check if `source` has `progenitor_app_id` set, so it can use that in the event that a copy is copied.